internal/pkg/db/mongo/export.go

```diff
@@ -122,11 +175,11 @@ func (mc MongoClient) getRegistrations(q bson.M) ([]export.Registration, error)
 }
 
 // Get a single registration for the passed query
-func (mc MongoClient) getRegistration(q bson.M) (export.Registration, error) {
+func (mc MongoClient) getRegistration(q bson.M) (models.Registration, error) {
```

Comment:
If you modify this function to return `models.Registration{}` explicitly when err != nil, you can simplify the calling code (`registrationById`, `registrationByName`).

Reply:
> If you modify this function to return models.Registration{} explicitly when err != nil, 

Do you mean when `err == nil`?

Reply:
No, I think he means return the empty struct when there's an error (err != nil).